pom.xml

```
@@ -1,9 +1,86 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.openmrs</groupId>
-  <artifactId>openmrs-content-hiv</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <name>Archetype - openmrs-content-hiv</name>
-  <url>http://maven.apache.org</url>
+	<modelVersion>4.0.0</modelVersion>
+ 	<groupId>org.openmrs.content</groupId>
+  	<artifactId>openmrs-content-hiv</artifactId>
+  	<version>1.0-SNAPSHOT</version>
+  	<name>OpenMRS HIV Content Package</name>		
+	<packaging>pom</packaging>
+	<developers>
+		<developer>
+			<name>OpenMRS</name>
+			<organization>OpenMRS</organization>
+			<organizationUrl>http://openmrs.org</organizationUrl>
+		</developer>
+	</developers>
+	<scm>
+		<connection>scm:git:git@github.com:openmrs/openmrs-content-hiv.git</connection>
+		<developerConnection>scm:git:git@github.com:openmrs/openmrs-content-hiv.git</developerConnection>
+		<url>https://github.com/openmrs/openmrs-content-hiv.git</url>
+		<tag>HEAD</tag>
+	</scm>	
+	<dependencies>
+		<dependency>
+            <groupId>org.openmrs.maven.plugins</groupId>
+            <artifactId>openmrs-packager-maven-plugin</artifactId>
+            <version>1.8.0-SNAPSHOT</version>
+        </dependency>
+	</dependencies>
+	<build>
+		<plugins>			
+			<plugin>
+				<groupId>org.openmrs.maven.plugins</groupId>
+				<artifactId>openmrs-packager-maven-plugin</artifactId>
+				<version>1.8.0-SNAPSHOT</version>
+				<executions>
+					<execution>
+						<id>validate-assemble</id>
+						<phase>package</phase>						
+						<configuration>
+							<sourceFile>{project.basedir}/content.properties</sourceFile>
+						</configuration>
+						<goals>
+						<goal>validate-assemble</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-assembly-plugin</artifactId>
+				<version>3.3.0</version>
```

We should always use the latest version available, if possible.

```suggestion
				<version>3.7.1</version>
```

---

Done!